mhctools/base_commandline_predictor.py

```diff
@@ -20,9 +20,7 @@
 
 from six import string_types
 from typechecks import require_string, require_integer, require_iterable_of
-from mhcnames import normalize_allele_name
-from mhcnames.parsing_helpers import AlleleParseError
-
+from mhcnames import normalize_allele_name, AlleleParseError
```

Comment:
@julia326 this needs to be e.g. `from mhcnames.parsing_helpers import AlleleParseError` to work well with your 0.1.0 fixing PR right?